src/System.Management.Automation/engine/CommonCommandParameters.cs

```diff
@@ -232,6 +232,26 @@ public string PipelineVariable
             set { _commandRuntime.PipelineVariable = value; }
```

Comment:
@KirkMunro, your last commit had 12 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

Experimental Feature Basic Tests - Feature-Enabled.Dynamic parameters - Function-Test-MyDynamicParamOne
```powershell
Expected 13, but got 12.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1: line 361
361:         $command.Parameters.Count | Should -Be ($CommonParameterCount + 1)
```
Experimental Feature Basic Tests - Feature-Enabled.Dynamic parameters - Cmdlet-Test-MyDynamicParamOneCSharp
```powershell
Expected 13, but got 12.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1: line 361
361:         $command.Parameters.Count | Should -Be ($CommonParameterCount + 1)
```
Experimental Feature Basic Tests - Feature-Enabled.Dynamic parameters - Function-Test-MyDynamicParamTwo
```powershell
Expected 13, but got 12.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1: line 361
361:         $command.Parameters.Count | Should -Be ($CommonParameterCount + 1)
```
Experimental Feature Basic Tests - Feature-Enabled.Dynamic parameters - Cmdlet-Test-MyDynamicParamTwoCSharp
```powershell
Expected 13, but got 12.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1: line 361
361:         $command.Parameters.Count | Should -Be ($CommonParameterCount + 1)
```
Experimental Feature Basic Tests - Feature-Enabled.Experimental parameter set - 'Test-MyRemoting' should have '-ComputerName' only
```powershell
Expected 13, but got 12.
at <ScriptBlock>, /Users/runner/runners/2.160.0/work/1/s/test/powershell/engine/ExperimentalFeature/ExperimentalFeature.Basic.Tests.ps1: line 319
319:         $command.Parameters.Count | Should -Be ($CommonParameterCount + 1)
```